plugins/reporters/static-html/src/main/kotlin/StaticHtmlReporter.kt

```diff
@@ -161,7 +161,7 @@ class StaticHtmlReporter : Reporter {
                         issueTable(tablesReport.advisorIssueSummary)
                     }
 
-                    tablesReport.projectDependencies.forEach { (_, table) ->
+                    tablesReport.projectDependencies.forEach { table ->
```

Comment:
Commit message: Maybe "from the previous commit" -> "due to the previous commit" (or "thanks to", "because of")